app/templates/image-set.hbs

```diff
@@ -19,8 +19,9 @@
   {{/if}}
 
   <div class='image-set-save-or-delete'>
-    {{#if showSaveButton}}
-      <button class='btn btn-primary btn-large' type='button' {{action 'saveImageSet'}}>
+    {{#if creatingNewImageSet}}
+      <button {{bind-attr class=':btn :btn-primary :btn-large enableShowButton::disabled'}}
+        type='button' {{action 'saveImageSet'}}>
```

Comment:
technically should also bind-attr `disabled`, though there is no CP available for it. Would be something like `{{bind-attr disabled=isShowButtonDisabled}}` where `isShowButtonDisabled` is `Ember.computed.not('enableShowButton')`